### PR TITLE
Do not append `\n` for PROCESS since that output has all of the delim…

### PR DESCRIPTION
**Pull request by dkashyn-sfdc**
…iters and can be chunked not per line.

# Checklist

- [x] I have filed an issue about this change and discussed potential changes with the maintainers.
- [ ] I have received the approval from the maintainers to make this change.
- [x] This is not a stylistic, refactoring, or cleanup change.

Please note that the maintainers will not be reviewing this change until all checkboxes are ticked. See 
the [Contributions](https://github.com/bazelbuild/intellij#contributions) section in the README for more 
details.

# Discussion thread for this change

Issue number: [7145](https://github.com/bazelbuild/intellij/issues/7145)

# Description of this change

### CODE REVIEW

File: base/src/com/google/idea/blaze/base/toolwindow/ConsoleView.java [hunk 1]
```diff
@@ -237,19 +237,17 @@ void println(PrintOutput output) {
...
     ansiEscapeDecoder.escapeText(
         text,
         outputType == OutputType.ERROR ? ProcessOutputTypes.STDERR : ProcessOutputTypes.STDOUT,
         (t, k) -> consoleView.print(t, ConsoleViewContentType.getConsoleViewType(k)));
-    consoleView.print(
-        "\n",
-        outputType == OutputType.ERROR
-            ? ConsoleViewContentType.ERROR_OUTPUT
-            : ConsoleViewContentType.NORMAL_OUTPUT);
+    if (outputType != OutputType.PROCESS) {
```
Comment (dkashyn-sfdc) [inline base/src/com/google/idea/blaze/base/toolwindow/ConsoleView.java:244]: Using separators "as is" for the process and adding potentially omitted ones for other output types.

**Review by LeFrosch — APPROVED** [review]
Thank you!

**Comment by tpasternak** [comment]
can we close this #7145 now?

**Comment by dkashyn-sfdc** [comment]
> can we close this #7145 now?

Yes!

**Pull request merged**